### PR TITLE
SHA256 support

### PR DESCRIPTION
**Pull request by dspartz**

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by muellerfabian** [comment]
👍   :)

**Comment by dspartz** [comment]
Who can merge?

File: pom.xml [hunk 1]
```diff
@@ -19,12 +19,12 @@
 
     <groupId>com.yahoo.sshd</groupId>
     <artifactId>sshd_proxy</artifactId>
-    <version>0.2.18-SNAPSHOT</version>
+    <version>0.2.18</version>
```
Comment (charlesk40) [inline pom.xml]: I think we have to the let maven release plugin to do this by doing "mvn release:prepare" after this pr has been merged which will create the tag with sshd_proxy-0.2.18 so travis can publish to bintray.

File: pom.xml [hunk 2]
```diff
@@ -19,12 +19,12 @@
...
     <packaging>jar</packaging>
 
     <scm>
         <developerConnection>scm:git:ssh://git@github.com/yahoo/artifactory_ssh_proxy.git</developerConnection>
-        <tag>HEAD</tag>
+        <tag>ssh_proxy-0.2.18</tag>
```
Comment (charlesk40) [inline pom.xml]: Same here

**Comment by charlesk40** [comment]
👍

**Pull request merged**